test-common/test-common-runtime/src/main/java/org/apache/kafka/common/test/KafkaClusterTestKit.java

```diff
@@ -556,7 +556,7 @@ public String bootstrapServers() {
             int brokerId = entry.getKey();
             BrokerServer broker = entry.getValue();
             ListenerName listenerName = nodes.brokerListenerName();
-            int port = broker.boundPort(listenerName);
+            int port = broker.boundPort(ListenerName.normalised(listenerName.value()));
```

Comment:
Could you please add comments to explain the behavior? Also, the controller listener name needs comment too.

Reply:
> the controller listener name needs comment too.

line#578